salt/states/gpfs.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+'''
+Manage GPFS Clusters
+========================
+
+Example:
+
+.. code-block:: yaml
+
+      gpfs.joined
+        - cluster: gpfs.cluster
+'''
+from __future__ import absolute_import
+
+# Import python libs
+import logging
+import os
+
+# Import salt libs
+import salt.utils
+
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    '''
+    Only load if GPFS is installed.
+    '''
+    os.environ["PATH"] += os.pathsep + '/usr/lpp/mmfs/bin'
+    return salt.utils.which('mmlscluster') is not None
```

Comment:
instead of using the same check as the module, you can just check `if 'gpfs.start_cluster' in __salt__:`